build.gradle

```diff
@@ -20,7 +20,7 @@ ext {
 
   // Shortcuts for libraries we are using
   libraries = [
-      grpc: 'io.grpc:grpc-all:0.9.0',
+      grpc: 'io.grpc:grpc-all:0.14.0',
```

Comment:
We shouldn't take this upgrade lightly. We would also need to make the dependency in our generated grpc maven packages match this.